runtime/major_gc.c

```diff
@@ -389,10 +383,10 @@ static void mark_slice (intnat work)
   value v;
   header_t hd;
   mlsize_t size, i, start, end; /* [start] is a local copy of [current_index] */
-#ifdef CAML_INSTR
-  int slice_fields = 0;
-#endif
-  int slice_pointers = 0; /** gcc removes it when not in CAML_INSTR */
+  #ifdef CAML_INSTR
+  int slice_fields = 0; /** eventlog counters */
+  #endif /*CAML_INSTR*/
```

Comment:
```suggestion
#ifdef CAML_INSTR
  int slice_fields = 0; /** eventlog counters */
#endif /*CAML_INSTR*/
```
We don't like to indent `#ifdef`s in this codebase...